components/service/glean/src/test/java/mozilla/components/service/glean/DispatchersTest.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean
+
+import kotlinx.coroutines.launch
+import org.junit.Test
+
+import org.junit.Assert.assertNotSame
+import org.junit.Assert.assertSame
+
+class DispatchersTest {
+
+    @Test
+    fun `API scope runs off the main thread`() {
+        val mainThread = Thread.currentThread()
+
+        Dispatchers.APIScope.launch {
+            assertNotSame(mainThread, Thread.currentThread())
+        }
```

Comment:
You may want to join() here. Otherwise this may not get executed before the end of the test. Maybe you even want to have some boolean var that you set to `true` inside so that you can verify that it did something at all.